generate_build_config.py

```diff
@@ -33,23 +34,43 @@
 - mv $CHROOT_ROOT/build/livecd.ubuntu-cpc.* /home/ubuntu/images
 """
 
+WRITE_FILES_TEMPLATE = """\
+write_files:
+- encoding: b64
```

Comment:
And I can't find the user-data metadata limit for Azure as that's the cloud used in the example.  Practically speaking I don't think their limit is all that low (16K maybe?).